src/util/test-helper.tsx

```diff
@@ -0,0 +1,41 @@
+import { BaseTheme } from "@lifesg/react-design-system";
+import { noop } from "lodash";
+import React from "react";
+import {
+    BuilderContext,
+    EBuilderMode,
+    IBuilderState,
+} from "src/context-providers";
+import { ThemeProvider } from "styled-components";
+
+export const mockBuilderState: IBuilderState = {
```

Comment:
usually we put the mocks below the main exportable. Can add a section header stating `MOCKS`

And also rename it to be `MOCK_BUILDER_STATE`

Reply:
sorry I need to interject on this, actually there is no need to export / spread `MOCK_BUILDER_STATE` in other tests because it will always be spread inside `TestHelper.withProviders()`, may I request to:
- not export `MOCK_BUILDER_STATE`
- remove using `MOCK_BUILDER_STATE` in all other tests

note: this is the same comment as https://github.com/LifeSG/web-form-builder/pull/29

Reply:
this is not updated?